doc/en/multitest.rst

```diff
@@ -173,20 +173,23 @@ Name customization
 ------------------
 
```

Comment:
Please first state the default behavior and make this part more brief. My suggestion:

By default, a testsuite is identified in the report by its class name, and testcase by function name. User can specify custom for testsuite and testcase like this: 

    * @testcase(name="My Testcase")
    * @testsuite(name="My Test Suite")
    * @testsuite(name=lambda cls_name, suite: "{} -- {}".format(cls_name, id(suite)))

Reply:
changed